### PR TITLE
Added Ipfs with TiddlyWiki

### PR DESCRIPTION
**Pull request by xmaysonnave**
## What kind of change is it?

- [X] Addition
- [ ] Removal
- [ ] Edit
- [ ] Other

### Checklist

- [X] I edited the `/data` directory instead of the [README.md](https://github.com/ipfs/awesome-ipfs/blob/master/README.md).
- [X] This PR includes only one addition/removal/edit.
- [X] I ran the `make build` command following my edits to the `/data` directory.
- [X] I have followed the [CONTRIBUTING.md guidelines](https://github.com/ipfs/awesome-ipfs/blob/master/CONTRIBUTING.md).

## Details

Provide details of your changes here.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Everyone is welcome to submit their new awesome-ipfs item. In order to add an el
 - [IPFS Drive](http://ipfs-drive.ydns.eu) - In browser file manager for IPFS. [Source](https://github.com/fazo96/ipfs-drive)
 - [IPFS Event Drop](https://github.com/travisperson/ipfs-event-drops) - An app that allows for improved visualization of ipfs events.
 - [IPFS ID and Public Key QR Codes Demo](https://ipfs.io/ipfs/zdj7Whr8X3zah99TSuyPjENaERcBW9C7B36EaCb1DEZ5pbbL9/) - A demo app that uses windows.ipfs to show you a QR Code of your IPNS Link and of your Public Key. [Source](https://github.com/ipfs-shipyard/demo-ipfs-id-qr-codes)
+- [Ipfs with TiddlyWiki](https://github.com/xmaysonnave/tiddlywiki-ipfs) - Ipfs with TiddyWiki is a TiddlyWiki plugin who aims to help developers, editors or users to save their wikis and attachments over Ipfs.
```
Comment (lidel) [inline README.md]:
```suggestion
- [IPFS with TiddlyWiki](https://github.com/xmaysonnave/tiddlywiki-ipfs) - IPFS with TiddyWiki is a TiddlyWiki plugin who aims to help developers, editors or users to save their wikis and attachments over IPFS.
```

File: data/apps.yaml [hunk 2]
```diff
@@ -356,3 +356,7 @@ content:
     source: https://github.com/bdchain/Minerva
     description: >
       Minerva is a storage plugin of Apache Drill that connects IPFS's decentralized storage and Drill's flexible query engine.
+  - title: Ipfs with TiddlyWiki
```
Comment (lidel) [inline data/apps.yaml]:
```suggestion
  - title: IPFS with TiddlyWiki
```

File: data/apps.yaml [hunk 3]
```diff
@@ -356,3 +356,7 @@ content:
...
+    source: https://github.com/xmaysonnave/tiddlywiki-ipfs
+    description: >
+      Ipfs with TiddyWiki is a TiddlyWiki plugin who aims to help developers, editors or users to save their wikis and attachments over Ipfs.
```
Comment (lidel) [inline data/apps.yaml]:
```suggestion
      IPFS with TiddyWiki is a TiddlyWiki plugin who aims to help developers, editors or users to save their wikis and attachments over IPFS.
```

**Comment by lidel** [comment]
@xmaysonnave in spare moment, please apply above suggestions (`Ipfs` → `IPFS`).

I was unable to apply them due to error:

>

**Comment by xmaysonnave** [comment]
@lidel I did my best here. Synchronized from upstream, update according to your suggestion and tried to push a new PR with the updates. However there are some side effects related to other topics (Cyber and IPSE). Give me your thoughts if I can improve the situation. Thanks for your effort.

**Pull request merged**